build.gradle

```diff
@@ -4,19 +4,24 @@ plugins {
 
 group 'org.testcontainers.cloud.turbomode.sample'
 version '1.0-SNAPSHOT'
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
```

Comment:
While we are at it, we can also add a JVM toolchain, then users don't even need an appropriate JDK:
https://docs.gradle.org/current/userguide/toolchains.html

I know there were issues around JDK8 for M1 though, @eddumelendez might know more. However, if we pull in the appropriate JDK via toolchain, it is also fine to user a more modern LTS version.

Reply:
the only vendors providing Java 8 for M1 are bellsoft, corretto and zulu. I know that Temurin is planning to release it this month, hopefully.